android-components/components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -790,6 +790,22 @@ sealed class ContentAction : BrowserAction() {
      * Updates whether the toolbar should be forced to expand or have it follow the default behavior.
      */
     data class UpdateExpandedToolbarStateAction(val sessionId: String, val expanded: Boolean) : ContentAction()
+
+    /**
+     * Updates the [ContentState] with the provided [tabId] to the appropriate priority based on any
+     * existing form data.
+     */
+    data class CheckForFormDataAction(val tabId: String, val containsFormData: Boolean) : ContentAction()
+
+    /**
+     * Lowers priority of the [tabId] after certain period of time
+     */
+    data class ClearHighPrioritySessionAction(val tabId: String) : ContentAction()
```

Comment:
nit: Not sure if from the name is easy to determine that the priority will be lowered after certain period of time.

Reply:
From the name it sounds a it will clear the high priority for the given session, `SetDefaultPriorityAfterTime` or something similar 😅

Reply:
Sounds good, will update the name